dockertest/environment.py

```diff
@@ -49,8 +54,12 @@ def get_selinux_context(path):
     :return: SELinux context as a string or None if not set
     """
     try:
-        # may include null-termination in string
-        return getxattr(path, 'security.selinux').replace('\000', '')
+        if 'selinux' in sys.modules:
+            # First list item is null-terminated string length
+            return selinux.getfilecon(path)[1]
```

Comment:
...because you have the selinux code in a `try`. So Ubuntu shouldn't care. I don't think you even need the conditional, you could probably just `try return selinux.etc()`. You don't need any of the `xattr` stuff, because this isn't actually running in Ubuntu. Does that make sense?